internal/nettest/psiphontunnel/psiphontunnel.go

```diff
@@ -0,0 +1,33 @@
+// Package psiphontunnel implements the psiphontunnel nettest.
+package psiphontunnel
+
+import (
+	"context"
+
+	"github.com/measurement-kit/engine/internal/nettest"
+	"github.com/measurement-kit/engine/internal/nettest/psiphontunnel/runner"
+	"github.com/measurement-kit/engine/internal/version"
+	"github.com/measurement-kit/engine/model"
+)
+
+// Config contains the psiphontunnel nettest configuration.
+type Config = runner.Config
+
+// NewNettest creates a new psiphontunnel nettest.
+func NewNettest(config Config) *nettest.Nettest {
+	return &nettest.Nettest{
+		TestName:        "psiphontunnel",
+		TestVersion:     "0.0.1",
+		SoftwareName:    "MKEngine",
+		SoftwareVersion: version.Version,
```

Comment:
I find it a bit unusual that it's the responsibility of the nettest specific `NewNettest` implementation to define the `SoftwareName` and `SoftwareVersion`.

Reply:
I agree! This is now #26